src/Pelagos/Bundle/AppBundle/Controller/UI/SideBySideController.php

```diff
@@ -98,8 +98,13 @@ public function getVersions(Request $request)
             $data['modifier'] = $submission->getModifier()->getLastName() .
                 ', ' . $submission->getModifier()->getFirstName();
             $data['modificationtimestamp'] = $submission->getModificationTimeStamp()->format('c');
+            $data['datasetstatus'] = Dataset::DATASET_STATUSES[$submission->getDatasetStatus()];
```

Comment:
I don't think displaying the submission's dataset status was part of this user-story

Reply:
Probably we can do it another user-story? or we can ask the PO to add in this story

Reply:
Ok, as per PO, removed for now.